### PR TITLE
Add sticky nav to extensions table

### PR DESCRIPTION
**Pull request by Sinetheta**
Okay, I admit that's it's just a ball of JS, but I thought as long as didn't pull in any libs you'd let it slide 😉 

screen shot 2016-06-04 at 1 50 34 pm

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+var stickyHeaderToggle = function() {
+  this.targetTop = this.targetTop || stickyTarget.getBoundingClientRect().top + document.body.scrollTop;
+  if(document.body.scrollTop >= this.targetTop) {
+    stickyClone.classList.remove('hide')
+  } else {
+    stickyClone.classList.add('hide')
+  }
+};
+
+window.addEventListener('scroll', stickyHeaderToggle);
```
Comment (jhawthorn) [inline script.js]: anything on scroll needs to be debounced

File: script.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    fakeHeader.style.cssText = document.defaultView.getComputedStyle(th, "").cssText;
+    fakeHeader.style.position = 'absolute';
+    fakeHeader.style.left = thRect.left - targetRect.left;
+    cloneRow.appendChild(fakeHeader);
+  });
+  document.body.appendChild(stickyHeader);
+  return stickyHeader;
+}
+
+var stickyTarget = document.querySelector('[data-sticky-header]');
```
Comment (jhawthorn) [inline script.js]: What if this doesn't exist? Maybe a `querySelectorAll` and a `forEach`

File: script.js (hunk 1 above)
Comment (Sinetheta) [inline script.js]: But then I'll get latency on that sweet stick/unstick. Without a perf showing jank I'll only go to the nearest animation frame 😛

File: script.js (hunk 2 above)
Comment (Sinetheta) [inline script.js]: If it doesn't exist wouldn't we want to know it's time to fix/remove this code?

Comment (mtomov) [inline script.js]: hah : )

**Pull request closed without merging**